doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -189,6 +189,21 @@ KRKNWK-14299: NRPA MAC address cannot be set in Zephyr
 Bluetooth LE
 ============
 
+.. rst-class:: v2-9-0-nRF54H20-rc1 v2-9-0 v2-8-0
+
+NCSDK-31528: Deadlock on sysworkq with ``tx_notify`` in Host
+  Blocking the system workqueue if the :kconfig:option:`CONFIG_BT_HCI_ACL_FLOW_CONTROL` Kconfig option is disabled can cause a deadlock in the Bluetooth Host when running out of buffers in the HCI commands pool.
+  The Bluetooth Host uses the system workqueue to complete processing of transmitted ACL data.
+  Thus, using any blocking API on the system workqueue blocks the Bluetooth Host.
+  If the HCI commands pool is empty, and a blocking Bluetooth Host API that uses the :c:func:`bt_hci_cmd_send_sync` function is called from any thread (including the system workqueue) and the system workqueue is blocked, the deadlock can occur.
```

Comment:
```suggestion
  If the :kconfig:option:`CONFIG_BT_HCI_ACL_FLOW_CONTROL` Kconfig option is disabled, and the HCI commands pool is empty, and a blocking Bluetooth Host API that uses the :c:func:`bt_hci_cmd_send_sync` function is called from any thread (including the system workqueue), and the system workqueue is blocked, the deadlock can occur.
```

Reply:
I added instead some bullet points to try to give a better view of this, please have a look!